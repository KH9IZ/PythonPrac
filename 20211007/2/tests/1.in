122.33, 1-0.2j
