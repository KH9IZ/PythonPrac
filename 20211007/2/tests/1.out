(121.33+0.2j)
